src/sentry/consumer.py

```diff
@@ -0,0 +1,60 @@
+from __future__ import absolute_import, print_function
+
+from batching_kafka_consumer import AbstractBatchWorker
+
+from django.conf import settings
+
+import sentry.tasks.store as store_tasks
+from sentry.utils import json
+
+
+class ConsumerWorker(AbstractBatchWorker):
```

Comment:
Note that I made a single consumer abstraction because I think the development/single-tenant/on-prem usecase likely only needs 1 process that does all of the work.

For our production needs we can run as many of these as we want, targeted at individual topics (see command runner below).